setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
"best practices" is a bit of a loaded term when talking about pypi, but its the best setup I could find among the ~ 20 packages I could find on this list https://github.com/audreyr/cookiecutter#a-pantry-full-of-cookiecutters